timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -356,6 +390,30 @@ services:
         tags:
           - server-request-context
         returns: GetCommitTimestampResponse
+      acquireNamedMinTimestampLeases:
```

Comment:
Do we need these single namespace endpoints? is it just for metrics or something? What would the world look like if there was only batch-multiclient API and you simply send a single client's stuff through that if that's what you're running?

Reply:
What I mean is a single client setup could just do:

```
Map.of("clientName", GetMinLeasedNamedTImestampRequests.of())
```